gridlab-d/gridlab-d
# Unbound memory fix

#### What's this Pull Request do?
Fixes the unbound memory allocation issue present in current GridLAB-D's load behavior.

#### Where should the reviewer start?
Confirm the behavior noted in #1225 is now fixed, and no longer causing out of memory issues.

#### How should this be tested?
Cross check a variety of cases which make use of both automatic and manual object ID specification.

#### What are the relevant issues?
Resolves #1225, Resolves #1228
